src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
+      `
+    : Icon
+
+  return <ChevronIcon name="chevronDown" size={12} color="black" />
+}
+
+const SelectFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const SelectBox = Box.extend`
+  cursor: pointer;
+`
+
+const SelectLabel = Label.extend`
+  cursor: pointer;
+`
+
+const Select = props => (
+  <SelectFlex justify="center" align="center">
```

Comment:
We'll probably need to add a lot of ARIA attributes to compensate for accessibility in this component – here's a good starting place for research: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

Reply:
@jxnblk, great feedback on the potential support of the ARIA attributes. I've created #152 issue as a follow-up item to this PR.